pkg/resources/resources.go

```diff
@@ -29,7 +27,7 @@ func ToUnstructured(resource metav1.Object) (*unstructured.Unstructured, error)
 }
 
 // ToProper returns the proper object representation of a resource.
```

Comment:
```suggestion
// ToTyped returns the proper object representation of a resource.
```
I like the terminology used in [client-go docs](https://github.com/kubernetes/client-go/tree/master/examples/dynamic-create-update-delete-deployment), i.e. "typed" rather than "proper".